s3/pkg/datastore/yig/tests/object_test.go

```diff
@@ -253,6 +253,17 @@ func (ys *YigSuite) TestMultipartUploadSucceed(c *C) {
 	n, err := objReader.Read(buf)
 	c.Assert(n, Equals, 0)
 	c.Assert(err, Equals, io.EOF)
+	// list the parts of multipart uploaded object.
+	listPartsReq := &pb.ListParts{
+		Bucket:   obj.BucketName,
+		Key:      obj.ObjectKey,
+		UploadId: mu.UploadId,
+	}
+
+	listPartsResp, err := yig.ListParts(context.Background(), listPartsReq)
+	c.Assert(err, Equals, nil)
```

Comment:
Can you put some logs here, to get the information about asserts

Reply:
@nguptaopensds thanks for your review, and what is the kind of log you mentioned above? as far as i know, if the assert fails, gocheck will print the err.